### ISSUE TITLE
Please destroy the QApplication singleton before creating a new QApplication instance.

### ISSUE THREAD

**Issue by LZHLZHOOO** [tpaviot/pythonocc-core#1363]
<img width="386" alt="4b96f0dc14cb857437a767d6ca03e5d" src="https://github.com/user-attachments/assets/0d78c006-285f-4dc3-b1db-208d3569d74a">
I want to upload a 3D model in STP format, but I'm encountering this error. Does anyone know what might be causing this issue? Please help.

**Comment by jnwalther** [comment]
I'm not 100% sure if this is the same issue, but I got the same error message when trying to open a display window for a second in a single Python session. It seems to be related to Pyside6. Other projects seem to have [similar issues](https://github.com/runesc/PPG/issues/7).

The solution given in that issue seems to do the trick for me, i.e. go to https://github.com/tpaviot/pythonocc-core/blob/master/src/Display/SimpleGui.py and do
```diff
-  app = QtWidgets.QApplication(sys.argv)
+  app = QtWidgets.QApplication.instance() or QtWidgets.QApplication(sys.argv)
```
in line 230.

I haven't tested it completely, so I can't say if this change breaks anything for the other Qt versions.

**Comment by tpaviot** [comment]
Fixed by #1381
